doc/library/tensor/basic.txt

```diff
@@ -427,7 +427,8 @@ TensorVariable
     you'll want to call.
 
 
-.. class:: _tensor_py_operators(object)
+.. autoclass:: _tensor_py_operators
+   :members:
```

Comment:
This should add the documentation of transfer() which is only in its docstring.  We can move the rest later.